Articles of Association.md

```diff
@@ -0,0 +1,66 @@
+# Articles of Association
+1. Name
+    1. The name of the association is "Feather Association"
+2. Purpose
+    1. The purpose of the association is
+        1. The promotion of feather
+        2. The development of feather
+        3. The distribution of feather and associated software and tools
+        4. The general development of modding Minecraft
+3. Membership
+    1. Anyone who is willing can be admitted as a Member through registration. 
+    2. Registration takes place on the association's registration form. 
+    3. The membership fee is determined by the General Assembly.
+4. General Assembly
+    1. The general meeting is the highest authority of the association.
+    2. The Annual General Meeting is held once a year before the end of the ______ month and is convened with at least three weeks' notice by email to members who are eligible to vote.
+    3. The public is eligible to attend the general meeting.
+    4. All members who are actively contributing towards the goals stipulated in section 2. and has notified the Board of Directors 8 days prior to the General Assembly are eligible to vote.
```

Comment:
This way only people who have contacted the Board of Directors 8 days prior are eligible to vote. Maybe we should shorten the time required and make board members automatically have voting rights or do they automatically contact them self?